From 4d607fb14af5a17b9eb74bcb0bd8f84e2ecafad8 Mon Sep 17 00:00:00 2001
From: Adriana Reus <adriana.reus@nxp.com>
Date: Mon, 11 Sep 2017 10:29:48 +0300
Subject: [PATCH 2591/5242] MLK-16442-0: clk: imx8: Remove
 imx_clk_divider2_scu duplicate declaration.

commit  f711d5b0981a57629206a77597f9c379dab7c73d from
https://source.codeaurora.org/external/imx/linux-imx.git

The imx_clk_divider2_scu is declared twice.
Remove the second occurrence.

(BuildInfo: SCFW 9e9f6ec6, IMX-MKIMAGE 0, ATF 0)

Signed-off-by: Adriana Reus <adriana.reus@nxp.com>
Reviewed by: Ranjani Vaidyanathan <Ranjani.vaidyanathan@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/clk/imx/clk-imx8.h |    3 ---
 1 file changed, 3 deletions(-)

diff --git a/drivers/clk/imx/clk-imx8.h b/drivers/clk/imx/clk-imx8.h
index 7a99760..f191fd4 100644
--- a/drivers/clk/imx/clk-imx8.h
+++ b/drivers/clk/imx/clk-imx8.h
@@ -20,9 +20,6 @@ struct clk *imx_clk_divider2_scu(const char *name, const char *parent_name,
 struct clk *imx_clk_divider3_scu(const char *name, const char *parent_name,
 			sc_rsrc_t rsrc_id, sc_ctrl_t gpr_id);
 
-struct clk *imx_clk_divider2_scu(const char *name, const char *parent_name,
-				sc_rsrc_t rsrc_id, sc_pm_clk_t clk_type);
-
 struct clk *clk_register_gate_scu(struct device *dev, const char *name,
 		const char *parent_name, unsigned long flags,
 		u8 clk_gate_scu_flags, spinlock_t *lock,
-- 
1.7.9.5

